roles/ffh.yanic/tasks/main.yml

```diff
@@ -24,14 +29,69 @@
   when: goversion is changed
 
 - name: Install path to go stuff
-  copy:
-    content: |
-      export PATH=$PATH:/usr/local/go/bin:$GOPATH/bin
-      export GOPATH=/opt/go
+  template:
+    src: go.sh.j2
     dest: /etc/profile.d/go.sh
     mode: 755
 
-# TODO: install yanic
+- name: Check if yanic is already installed
+  stat:
+    path: "{{ yanic_source_path }}"
+  register: yanic_source_dir
+
+- name: Install yanic if not yet installed
+  command: "{{ go_bin_path }}/go install yanic"
+  when: not yanic_source_dir.stat.exists
+
+- name: Update to latest master
```

Comment:
Do we really want bleeding edge?

Reply:
That was my thought too: Latest version with "old" configuration?

Does Go have a dependency on rsync or what is it for?

Reply:
I read through yanics codebase and I don't think it's worth it to use their latest release.
It's from 2016, called `0.0.2` and 56 commits behind master.

The only considerable alternative would be to have a fixed commit, which we could do; I'm just no fan.
I'll open an issue t them asking for an appropriate solution; like a moving tag.

I don't expect many config changes anymore, they got pretty reliable a while ago.

Lastly the rsync path is a dependency for the ansible.synchronization module.
We use it to easily add and remove patch files on the remote.

Reply:
resolved in f4b4980455ac40ee8f9454d09e5ee6b64028eab6